lib/acts_as_paranoid/core.rb

```diff
@@ -112,10 +118,14 @@ def destroy
           end
         end
       else
-        destroy!
+        destroy_fully!
       end
     end
 
+    def destroy
+      destroy!
```

Comment:
@zzak I'm sorry, I haven't noticed that method.
@mvz Is there any reason why this is implemented like that?

Is there anything that I can help with? Thank you all.